stats/parsing-stats/lang/ruby/projects.txt

```diff
@@ -17,8 +17,8 @@ https://github.com/travis-ci/travis-tasks
 
 https://github.com/OWASP/railsgoat
 
-# TODO? skip? may contain many test files?
-https://github.com/ruby/ruby
+# tends to fail
```

Comment:
This is what I wrote for skipping the ocaml compiler in ../lang/ocaml/projects.txt:

We currently skip the compiler repo because it contains lots of test files
which are not representative of regular OCaml code

we could write a similar comment here.